operator/v1/types_ingress.go

```diff
@@ -1534,7 +1534,7 @@ type IngressControllerTuningOptions struct {
 	// subsequent reload.
 	//
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(s|m))+)$
+	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(ms|s|m))+)$
```

Comment:
It's probably safer to just support all possible suffixes
```suggestion
	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(ns|us|µs|ms|s|m|h))+)$
```